spec/integrations/disk_spec.rb

```diff
@@ -0,0 +1,56 @@
+describe 'datadog::disk' do
+  expected_yaml = <<-EOF
+  init_config: {}
+
+  instances:
+    - use_mount: 'yes'
+      excluded_filesystems:
+        - tmpfs
+      excluded_disks:
+        - /dev/sda1
+        - /dev/sda2
+      excluded_disk_re: '/dev/sde.*'
+      tag_by_filesystem: 'no'
+      excluded_mountpoint_re: '/mnt/somebody-elses-problem.*'
+      all_partitions: 'no'
+  EOF
+
+  cached(:chef_run) do
+    ChefSpec::SoloRunner.new(step_into: ['datadog_monitor']) do |node|
+      node.automatic['languages'] = { 'python' => { 'version' => '2.7.2' } }
+      node.set['datadog'] = {
+        api_key: 'someapikey',
+        disk: {
+          instances: [
+            {
+              use_mount: 'yes',
+              excluded_filesystems: ['tmpfs'],
+              excluded_disks: ['/dev/sda1', '/dev/sda2'],
+              excluded_disk_re: '/dev/sde.*',
+              tag_by_filesystem: 'no',
+              excluded_mountpoint_re: '/mnt/somebody-elses-problem.*',
+              all_partitions: 'no'
```

Comment:
Even though the disk check appears to correctly handle strings for boolean parameters, it's recommended to set them as actual booleans. Could you use booleans for `use_mount`, `tag_by_filesystem` and `all_partitions`? (and update the example you provided in the recipe accordingly).